dtfs-central-api/src/helpers/get-latest-tfm-facility-values.ts

```diff
@@ -4,32 +4,29 @@ import { TfmFacilitiesRepo } from '../repositories/tfm-facilities-repo';
 import { NotFoundError } from '../errors';
 import { convertTimestampToDate } from './convert-timestamp-to-date';
 import { getEffectiveCoverEndDateAmendment } from './amendments/get-effective-cover-end-date-amendment';
+import { LatestTfmFacilityValues } from '../types/tfm/tfm-facility';
 
 /**
  * Gets the latest values for the TFM facility with the supplied facility id
  * @param facilityId - The facility id
  * @param reportPeriod - The report period
  * @returns The latest values
  */
-export const getLatestTfmFacilityValues = async (
-  facilityId: string,
-  reportPeriod: ReportPeriod,
-): Promise<{
-  coverEndDate: Date;
-  coverStartDate: Date;
-  dayCountBasis: number;
-  interestPercentage: number;
-  coverPercentage: number;
-}> => {
+export const getLatestTfmFacilityValues = async (facilityId: string, reportPeriod?: ReportPeriod): Promise<LatestTfmFacilityValues> => {
```

Comment:
This function name needs updating. If not report period is passed in it doesn't return the latest values at all, it returns the initial values at facility creation, and if a report period is passed in it returns the values that are in effect at the end of that report period.